{
  "uid" : "30ccfa5c6b88ba97",
  "name" : "test_app_article_1_python",
  "fullName" : "scripts.test06_app_article.TestAppArticle#test_app_article_1_python",
  "historyId" : "246a1f0d7183ac2ee7850f9f4653813d",
  "time" : {
    "start" : 1600852011917,
    "stop" : 1600852041423,
    "duration" : 29506
  },
  "status" : "passed",
  "flaky" : false,
  "newFailed" : false,
  "beforeStages" : [ {
    "name" : "_UnitTestCase__pytest_class_setup",
    "time" : {
      "start" : 1600851986632,
      "stop" : 1600852011916,
      "duration" : 25284
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "shouldDisplayMessage" : false,
    "attachmentsCount" : 0,
    "hasContent" : false,
    "stepsCount" : 0
  } ],
  "testStage" : {
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ {
      "uid" : "3cb174bf05e23361",
      "name" : "log",
      "source" : "3cb174bf05e23361.txt",
      "type" : "text/plain",
      "size" : 3340
    } ],
    "parameters" : [ ],
    "shouldDisplayMessage" : false,
    "attachmentsCount" : 1,
    "hasContent" : true,
    "stepsCount" : 0
  },
  "afterStages" : [ {
    "name" : "_UnitTestCase__pytest_class_setup::0",
    "time" : {
      "start" : 1600852041424,
      "stop" : 1600852042939,
      "duration" : 1515
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "shouldDisplayMessage" : false,
    "attachmentsCount" : 0,
    "hasContent" : false,
    "stepsCount" : 0
  } ],
  "labels" : [ {
    "name" : "parentSuite",
    "value" : "scripts"
  }, {
    "name" : "suite",
    "value" : "test06_app_article"
  }, {
    "name" : "subSuite",
    "value" : "TestAppArticle"
  }, {
    "name" : "host",
    "value" : "DESKTOP-DGCGN8L"
  }, {
    "name" : "thread",
    "value" : "11956-MainThread"
  }, {
    "name" : "framework",
    "value" : "pytest"
  }, {
    "name" : "language",
    "value" : "cpython3"
  }, {
    "name" : "package",
    "value" : "scripts.test06_app_article"
  }, {
    "name" : "resultFormat",
    "value" : "allure2"
  } ],
  "parameters" : [ ],
  "links" : [ ],
  "hidden" : false,
  "retry" : false,
  "extra" : {
    "severity" : "normal",
    "retries" : [ {
      "uid" : "f37a2e7917c364e4",
      "status" : "broken",
      "statusDetails" : "selenium.common.exceptions.TimeoutException: Message:",
      "time" : {
        "start" : 1600851949250,
        "stop" : 1600851979875,
        "duration" : 30625
      }
    }, {
      "uid" : "58635a7dfd0b0a33",
      "status" : "broken",
      "statusDetails" : "urllib3.exceptions.MaxRetryError: HTTPConnectionPool(host='127.0.0.1', port=4723): Max retries exceeded with url: /wd/hub/session (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001CF48842CD0>: Failed to establish a new connection: [WinError 10061] 由于目标计算机积极拒绝，无法连接。'))",
      "time" : {
        "start" : 1600851421533,
        "stop" : 1600851421533,
        "duration" : 0
      }
    }, {
      "uid" : "6ef25392d7c420",
      "status" : "broken",
      "statusDetails" : "urllib3.exceptions.MaxRetryError: HTTPConnectionPool(host='127.0.0.1', port=4723): Max retries exceeded with url: /wd/hub/session (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001CF48796A30>: Failed to establish a new connection: [WinError 10061] 由于目标计算机积极拒绝，无法连接。'))",
      "time" : {
        "start" : 1600851411365,
        "stop" : 1600851411365,
        "duration" : 0
      }
    }, {
      "uid" : "659565dfd7eef747",
      "status" : "broken",
      "statusDetails" : "urllib3.exceptions.MaxRetryError: HTTPConnectionPool(host='127.0.0.1', port=4723): Max retries exceeded with url: /wd/hub/session (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001CF48973460>: Failed to establish a new connection: [WinError 10061] 由于目标计算机积极拒绝，无法连接。'))",
      "time" : {
        "start" : 1600851406216,
        "stop" : 1600851406216,
        "duration" : 0
      }
    }, {
      "uid" : "cbb1ed5b29da9506",
      "status" : "broken",
      "statusDetails" : "urllib3.exceptions.MaxRetryError: HTTPConnectionPool(host='127.0.0.1', port=4723): Max retries exceeded with url: /wd/hub/session (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x00000261663DFF10>: Failed to establish a new connection: [WinError 10061] 由于目标计算机积极拒绝，无法连接。'))",
      "time" : {
        "start" : 1600768397527,
        "stop" : 1600768397527,
        "duration" : 0
      }
    } ],
    "categories" : [ ],
    "tags" : [ ]
  },
  "source" : "30ccfa5c6b88ba97.json",
  "parameterValues" : [ ]
}